backend/Makefile

```diff
@@ -133,6 +135,7 @@ update-data: \
 	github-update-related-organizations \
 	github-update-users \
 	owasp-aggregate-projects \
+	owasp-sync-official-project-levels \
```

Comment:
The logic behind this target should be a part of `github-update-owasp-organization`